.github/workflows/code_metrics_pr.yml

```diff
@@ -40,6 +40,18 @@ jobs:
           publish_dir: ./public
           keep_files: true
 
+      - name: Comment PR
+        if: ${{ github.event.action == 'opened' }}
+        uses: thollander/actions-comment-pull-request@master
+        with:
+          message: |
+            ### :bar_chart: Code Metrics Report
+            Branch | Metrics
+            ------------ | -------------
+            This PR | [code-metrics.html](https://tier4.github.io/obstacle_stop_planner_refine/pr-check/pr${{ steps.plum.outputs.PULL_NUMBER }}/code-metrics.html)
+            main | [code-metrics.html](https://tier4.github.io/obstacle_stop_planner_refine/pr-check/main/code-metrics.html)
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
Could you follow these comments, please? :smile: 
https://github.com/tier4/obstacle_stop_planner_refine/pull/16#discussion_r599390686
https://github.com/tier4/obstacle_stop_planner_refine/pull/16#discussion_r599388240

Reply:
Fixed at fcf0ffb

Reply:
Thanks!